common-compose.yml

```diff
@@ -0,0 +1,68 @@
+wordpress:
+  build: .
+  privileged: true
+  env_file: _env
+
+nfs:
+    image: autopilot-nfs
```

Comment:
No, that is the same image, I just pulled the repo and built it locally. I was thinking this would change once this image gets it's permanent home on Docker hub or whatever.